README.md

```diff
@@ -40,10 +40,14 @@ addressing a single deprecation at a time, and prevents backsliding
 The initial steps needed to get started:
 
 1. Install the ember-cli-deprecation-workflow addon (`ember install ember-cli-deprecation-workflow`).
-2. Run your test suite\* with `ember test --server`.
-3. Navigate to your tests (default: http://localhost:7357/)
-4. Run `deprecationWorkflow.flushDeprecations()` from your browsers console.
-5. Copy the string output into `config/deprecation-workflow.js` in your project.
+3. Run your test suite\* with `ember test --server`.
```

Comment:
Above these steps there is a compatibility section, as at https://github.com/mixonic/ember-cli-deprecation-workflow?tab=readme-ov-file#getting-started. This PR seems like a good place to add a section for 3.0.

The contents of the compatibility docs would want to match the proposal at https://github.com/mixonic/ember-cli-deprecation-workflow/issues/169 👍